db/index.js

```diff
@@ -1,79 +1,93 @@
+    });
   });
 });
 
 // insert into table_name (cols) values (vals)
 const addItem = (item) => new Promise((resolve, reject) => {
-  const {picture1, picture2, picture3, picture4, picture5, picture6, description} = item;
-  const pictureArray = [picture1, picture2, picture3, picture4, picture5, picture6];
-  const sql1 = `INSERT INTO descriptions(description) VALUES(?)`;
-  const sql15 = `SELECT LAST_INSERT_ID()`;
-  const sql2 = `INSERT INTO pictures (url, description_id) VALUES (?, ?)`;
-  connection.query(sql1, [description], (err, results) => {
-    if (err) {
-      return reject(err);
-    }
-    connection.query(sql15, (err, results) => {
+  const {
+    picture1,
+    picture2,
+    picture3,
+    picture4,
+    picture5,
+    picture6,
+    description
+  } = item;
+  const queryArray = [
+    description,
+    picture1,
+    picture2,
+    picture3,
+    picture4,
+    picture5,
+    picture6
+  ];
+  const sql1 = `INSERT INTO pictures_service (
+    description,
+    picture1,
+    picture2,
+    picture3,
+    picture4,
+    picture5,
+    picture6
+  ) VALUES (?)`;
+  client.connect().then(() => {
+    client.query(sql1, queryArray, (err, results) => {
       if (err) {
+        client.end();
         return reject(err);
       }
-      const lastInsertId = results[0]['LAST)_INSERT_ID()'];
-      for (var i = 0; i < pictureArray.length - 1; i++) {
-        connection.query(sql2, [pictureArray[i], results[0]['LAST_INSERT_ID()']], (err, results) => {
-          if (err) {
-            return reject(err);
-          }
-          return resolve(results);
-        });
-      }
+        client.end();
+        return resolve(results);
     });
   });
 });
 
 // update table_name set col=val where col=val
 const updateItem = (item) => new Promise((resolve, reject) => {
-  const {table, id, update} = item;
-  const field = table === 'descriptions' ? 'description' : 'url';
-  const sql1 = `UPDATE ${table} SET ${field}="${update}" WHERE id=${id}`;
-  console.log(sql1);
-  connection.query(sql1, (err, results) => {
-    if (err) {
-      return reject(err);
-    }
-    return resolve(results);
+  const {field, id, update} = item;
+  const sql2 = `UPDATE pictures_service SET ${field}='${update}' WHERE id=${id}`;
+  client.connect().then(() => {
+    client.query(sql2, (err, results) => {
+      if (err) {
+        client.end();
+        return reject(err);
+      }
+      client.end();
+      return resolve(results);
+    });
   });
 });
 
-// below is just copied, but represents the delete function
 // update to delete from table_name where col=val
 const deleteItem = (id) => new Promise((resolve, reject) => {
-  const sql1 = `DELETE FROM descriptions WHERE id=${id}`;
-  const sql2 = `DELETE FROM pictures WHERE description_id=${id}`;
-  connection.query(sql2, (err, results) => {
-    if (err) {
-      return reject(err);
-    }
-    connection.query(sql1, (err, results) => {
+  const sql13 = `DELETE FROM pictures_service WHERE id=${id}`;
```

Comment:
You might consider, for clarity's sake, naming these something like 'deleteQuery' or what have you instead of  'sql + int'. I tend to